UPGRADE-4.3.md

```diff
@@ -133,15 +133,15 @@ Security
 
    After:
    ```php
-   protected function getState(): array
+   protected function __serialize(): array
```

Comment:
The methods are going to be public in PHP, so we should do the same here and probably mark the methods as `@internal` for now (that's what I've done in the Mime component).

Reply:
actually, we cannot really make them internal when the class allows extensibility by inheritance, because that's the only way to add extra data to serialized payloads

Reply:
Just updated all the forgotten `protected` to `public` 😅